crates/swc_css_ast/src/selector.rs

```diff
@@ -197,34 +197,32 @@ pub struct PseudoElementSelector {
 
 #[ast_node]
 pub enum PseudoSelectorChildren {
-    #[tag("Nth")]
-    Nth(Nth),
+    #[tag("AnPlusB")]
+    AnPlusB(AnPlusB),
```

Comment:
Isn't `AnPlusBNotation` better here? The current type definition will have multiple representations of a single value.


In this case, it's `PseudoSelectorChildren::Ident` and `PseudoSelectorChildren::AnPlusB(AnPlusB::Ident)`

Reply:
I found that this is quite annoying while working on es transforms, due to `PatOrExpr`

Reply:
@kdy1 Yep, I am thinking about it too, but here interesting case:
- `PseudoSelectorChildren::Ident` will have another values, for example `p:lang(en) {}`, `en` will be it
- `PseudoSelectorChildren::AnPlusB(AnPlusB::Ident)` is `even`/`odd`

It is not different in AST, but it is different on parser/visitor side, so if I want to visit all `AnPlusB` nodes (i.e. `Ident` or `AnPlusBNotation`), I will use `visit_an_plus_b` (same for `VisitMut`), you can see it in our minifier.

Also if somebody want to parse `an+b` notation, this will be used:
```rs
let an_plus_b: AnPlusB = parse_str(&"even", ParserConfig { ..Default::default() }, &mut errors).unwrap();
```

That is why we have two idents here.